mao-images.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+pod=$(oc get pod -n openshift-machine-api | grep '^machine-api-operator-')
+
+if [[ -z "$pod" ]]; then
+    echo "ERROR: No machine-api-operator pod found in openshift-machine-api namespace" 1>&2
+    exit 1
+fi
+
+oc exec pod/$pod -n openshift-machine-api -c machine-api-operator -- cat /etc/machine-api-operator-config/images/images.json
```

Comment:
One downside of this is that once you scale the MAO down, it doesn't work any more. An alternative would be to do:

```
oc get configmap -n openshift-machine-api machine-api-operator-images -o jsonpath="{.data.images\.json}"
```

That gets the content of the configmap provided by the CVO directly (rather than reading it from where it is mounted within the pod).

Reply:
Good point, done.